liwords-ui/src/App.tsx

```diff
@@ -96,6 +109,29 @@ const App = React.memo(() => {
     }
   }, [getFullBlocks, pendingBlockRefresh]);
 
+  const getMods = useCallback(() => {
+    axios
+      .post<ModsResponse>(
+        toAPIUrl('user_service.SocializeService', 'GetModList'),
+        {},
+        { withCredentials: true }
```

Comment:
this particular call does not require credentials (so that anon users can also see who's a moderator/admin).